runtime-light/component/component.cpp

```diff
@@ -51,12 +137,6 @@ void ComponentState::process_platform_updates() noexcept {
         }
       } else { // update on incoming stream
         php_debug("got new incoming stream %" PRIu64, stream_d);
-        if (standard_stream_ != INVALID_PLATFORM_DESCRIPTOR) {
```

Comment:
Is there any guarantee that `k2-oneshot` component will not get second incoming stream?

Reply:
It's guaranteed by the K2 platform